internal/services/codesigning/trusted_signing_account_data_source_test.go

```diff
@@ -41,12 +41,12 @@ provider "azurerm" {
 
 resource "azurerm_resource_group" "test" {
   name     = "acctestRG-%[1]d"
-  location = "%[3]s"
+  location = "eastus" # Hardcoding location because resource type Microsoft.CodeSigning/codeSigningAccounts is only available in 'westcentralus, westus2, northeurope, westeurope, eastus, westus, westus3, centraluseuap'
```

Comment:
This should be fixed in `settings.kt` via override, rather than hard-coding per test.